website/index.html

```diff
@@ -56,8 +56,8 @@ <h2>Get started now</h2>
       <div class="steps">
         <div class="step step-1">
           <!--label>Step 1</label-->
-          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/cloud-ops-sandbox.git&cloudshell_git_branch=v0.3.0&cloudshell_working_dir=terraform&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image:v0.3.0"
-             target="_blank">Open in Google Cloud Shell</a>
+          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/cloud-ops-sandbox.git&cloudshell_git_branch=v0.3.0&cloudshell_working_dir=terraform&shellonly=true&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image:v0.3.0&cloudshell_tutorial=tutorial.md"
+             target="_blank" onclick="return confirm('Cloud Operations Sandbox will create a new GCP project and will take ~15-20 minutes to complete. Are you sure you want to continue?')">Open in Google Cloud Shell</a>
```

Comment:
Looks like you're adding back in the pop-up. Simon asked to remove it since it can cause issues with pop-up blockers

Reply:
Oops, I did not know that! I will un-do that.